databox/api/src/Metadata/MetadataNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Metadata;
+
+use PHPExiftool\Driver\Metadata\Metadata;
+use PHPExiftool\Driver\Metadata\MetadataBag;
+use PHPExiftool\Driver\Value\Binary;
+
+class MetadataNormalizer
+{
+    /**
+     * normalize metadata from metadataManipulator bundle (for File.metadata).
+     */
+    public function normalizeToArray(MetadataBag $bag): array
```

Comment:
I would not precise `toArray` here, also there is no other way to normalize.

```suggestion
    public function normalize(MetadataBag $bag): array
```